src/index.ts

```diff
@@ -5,12 +5,12 @@ import {
 	RouteBases,
 	Routes,
 	ComponentType,
-	ButtonStyle,
 	TextInputStyle,
 } from "discord-api-types/v10";
 import { createCommands } from "./commands";
+import { sendModalMessage, deleteMessage } from "./messages";
 import { GetQuestionStateFromCommandName, NeedsMoreInfo, Unanswered } from "./question-state";
-import { getState, setState } from "./state";
+import { getKVState, setKVState, DOState, getDOState, setDOState, deleteDOState } from "./state";
```

Comment:
I'm thinking having both a KV/DO is overkill for this - let's just use one joint state. Given all of the requests come from the same colo, it's pretty safe to use KV, but a DO is also fine - not too bothered